DashboardTests/ConnectionTest.cs

```diff
@@ -99,10 +90,7 @@ public void TestConnectionPageChangeConnectionType()
         });
 
         // Change Connection Type to IP Address
-        window.MouseDown(new Point(370, 170), MouseButton.Left);
-        window.MouseUp(new Point(370, 170), MouseButton.Left);
-        window.MouseDown(new Point(360, 220), MouseButton.Left);
-        window.MouseUp(new Point(360, 220), MouseButton.Left);
+        connectionType.SelectedItem = "IP Address";
```

Comment:
How come this one is IP Address now?

Reply:
it tried to click on ip address in the drop down and i didnt want to work out the coordinates, so i just set the value

Reply:
fair